combat-trainer.lic

```diff
@@ -450,9 +475,14 @@ class LootProcess
       @last_rites_timer = Time.now
       return
     end
+
+    @equipment_manager.stow_weapon(game_state.whirlwind_offhand_name) if game_state.currently_whirlwinding?
```

Comment:
GameState has a copy of equipment manager. I suspect this line could be moved there and be a bit simpler. What do you think?